core/domain/story_domain_test.py

```diff
@@ -92,6 +93,15 @@ def test_defaults(self):
         }
         self.assertEqual(story.to_dict(), expected_story_dict)
 
+    def test_get_acquired_skill_ids_for_node_ids(self):
```

Comment:
Not enough test cases. What if multiple nodes have overlapping skill IDs? What if some nodes have more than one acquired skill ID? What if there are no acquired skill IDs? Test all these. 

Pretend someone else wrote the implementation for this function and you can't see the code they wrote, but you're responsible for making sure that function is fully correct in all cases. Write tests with that mindset.

Reply:
Done.